kedro-training/docs/01_prerequisites.md

```diff
@@ -1,3 +1,53 @@
+# Training pre-requisites
+This Kedro training materials assume the following pre-requisites for your technical understanding.
+
+We assume understanding of these technologies and concepts during the training. In order to maximise your learning experience from the Kedro training, please review them before going through the training (there are external resource links for you to get a refresher on each topic).
+
+- Intermediate Python knowledge
+  - [Basic Python](https://developers.google.com/edu/python)
+    - Functions, loops, conditional statements
+    - Common data structures: lists, dictionaries, tuples
+    - [IO operation](https://realpython.com/working-with-files-in-python/) (reading and writing data)
```

Comment:
do we actually use I/O operations anywhere in the tutorial (trying to remember), do we need to specify this? if we do, can we link to python I/O docs instead?

Reply:
I think they should know how to read/write data with Python even though we won't be touching directly, since all the datasets do this underneath. I moved IO operation in `Functions, loops, conditional statements, IO operration`, and removed the link :)